dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -173,18 +193,29 @@ public fun DataFrame.Companion.readExcel(
     sheetName: String? = null,
     skipRows: Int = 0,
     columns: String? = null,
+    formattingOptions: FormattingOptions? = null,
     rowsCount: Int? = null,
     nameRepairStrategy: NameRepairStrategy = NameRepairStrategy.CHECK_UNIQUE,
 ): AnyFrame {
     val sheet: Sheet = sheetName
         ?.let { wb.getSheet(it) ?: error("Sheet with name $sheetName not found") }
         ?: wb.getSheetAt(0)
-    return readExcel(sheet, columns, skipRows, rowsCount, nameRepairStrategy)
+    return readExcel(sheet, columns, formattingOptions, skipRows, rowsCount, nameRepairStrategy)
+}
+
+/**
+ * @param range comma separated list of Excel column letters and column ranges (e.g. “A:E” or “A,C,E:F”)
+ */
+public class FormattingOptions(range: String, public val formatter: DataFormatter = DataFormatter()) {
```

Comment:
I do have one concern exposing `DataFormatter` to the public API as that makes the excel implementation stuck to the `org.apache.poi.ss.usermodel` package which is not great if we want to swap excel implementations

Reply:
Sort of, yes. Thing is that Sheet is already a part of public API. I thought potential new implementation could live alongside existing one

Reply:
I'll make this dataFormatter argument only for overload with Sheet parameter

Reply:
that may be better indeed, to limit how much we expose. We did the same with Klaxon and JSON some time back.
I wouldn't have two Excel implementations side by side though. Best to keep it simple for the user and just have 1 `DataFrame.readExcel()` implementation